src/crypto/tls/handshake_client_tls13.go

```diff
@@ -36,6 +36,8 @@ type clientHandshakeStateTLS13 struct {
 	transcriptInner hash.Hash
 	masterSecret    []byte
 	trafficSecret   []byte // client_application_traffic_secret_0
+
+	handshakeTimings *EXP_EventClientHandshakeTimingInfo
```

Comment:
I would put this on the stack rather than the heap. (I.e., don't make this a pointer.)